src/matrixrtc/LivekitTransport.ts

```diff
@@ -0,0 +1,46 @@
+/*
+Copyright 2023 New Vector Ltd
```

Comment:
👋 welcome to the future! Please update your copyrights as you step outside the time machine :)